src/index.js

```diff
@@ -311,6 +311,7 @@ module.exports = function (options, callback) {
       return msg
     }
     debuglogger(msg)
+    return ''
```

Comment:
I see, this is an old code path that hasn't really been maintained.

Reply:
To make a little bit more sense, please return `msg` instead. Also feel free to do an `if/else` with the `console.error/debuglogger` and put the (shared) `return msg` after the `if/else`. I.e.

Reply:
but msg is not allowed to be more than "". If I use msg, there is always content in strErr. I would still need to set msg = ''

Reply:
> but msg is not allowed to be more than ""

Reply:
Better to have it as you had it and just return `` when no error. Sorry I didn't realise exactly what you were trying to fix before.